src/exe.ml

```diff
@@ -20,6 +20,13 @@ module Linkage = struct
     ; flags : string list
     }
 
+  module Js = struct
+    type linkage = t
+    type t =
+      | Js
+      | NonJs of linkage
```

Comment:
Our naming convention is `Foo_bar` instead of `FooBar` btw.

Reply:
Thanks, fixed.